design/v6-bundle/project/styles_v3.css

```
@@ -0,0 +1,90 @@
+
+:root {
+  --bg: #0b0b0f;
+  --bg-soft: #13131a;
+  --ink: #ededf2;
+  --ink-soft: #9a9aa8;
+  --ink-mute: #5a5a68;
+  --line: #23232d;
+}
+
+* { box-sizing: border-box; }
+html, body { margin: 0; background: var(--bg); color: var(--ink); font-family: 'Inter', system-ui, sans-serif; -webkit-font-smoothing: antialiased; }
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)